### DISCUSSION TITLE
Styling for "menuselection" is missing

### DISCUSSION THREAD

**Discussion by sscherfke** [pradyunsg/furo#601] [Potential Issue]
Furo does currently not provide styling for the Sphinx role [menuselection](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection).

I tried using the `.guilabel` style for `.menuselection` in a custom css and was quite happy with it. :)

**Comment by Kangie** [comment]
@pradyunsg 

`menuselection` is a [default sphinx role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection). `sphinx_rtd_theme` [includes styling](https://github.com/readthedocs/sphinx_rtd_theme/blob/5a263753d52c1628c88392fbf52c729f5a8e79b5/src/sass/_theme_rst.sass#L587) which makes this role easy to distinguish from any surrounding text.

You include `menuselection` as an example in your [kitchen sink](https://pradyunsg.me/furo/kitchen-sink/generic/#keys-menu-labels) ([source](https://github.com/pradyunsg/furo/blob/426ea05d879ee7dfdf74ba7c8b089f73081abbfd/docs/kitchen-sink/generic.rst#keys--menu-labels)), however as identified here there's no specific style for it. When we rendered the example into HTML we end up with content that is not visually distinct from other elements in the paragraph around it / the rest of the body text.

Given that the content is formatted to include "triangle arrow" characters by sphinx, perhaps styling it similarly to `kbd` would be appropriate?

```
  background-color: var(--color-background-secondary);
  border: 1px solid var(--color-foreground-border);
  border-radius: .2rem;
  box-shadow: 0 .0625rem 0 rgba(0,0,0,.2),inset 0 0 0 .125rem var(--color-background-primary);
  color: var(--color-foreground-primary);
  display: inline-block;
  font-size: var(--font-size--small--3);
  margin: 0 .2rem;
  padding: 0 .2rem;
  vertical-align: text-bottom;
  ```
  
Thanks for your work, the theme is lovely!
